### PR TITLE
Update AZ400_M05_Configuring_Agent_Pools_and_Understanding_Pipeline_S…

### PR DESCRIPTION
**Pull request by MOC-Labs-Review**
…tyles.md

**Replace issue name M00-LAB00:QUICK_DESCRIPTION, for example "M01-LAB01: My new issue" (or same name as linked Issue)**

## Related Issue

**Link related Github Issue** 🢂 Fixes # . (Include issue number after #)

## Checklist 
Mark completed with "x" between brackets, "[x]":
- [ ] Has related GitHub Issue 💥 [Create Issue](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#reporting-issues) 📝
- [ ] Tested it
- [ ] Read the PR collaboration guide 👓 [Collaboration Guide](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#pull-requests) 📝

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by MOC-Labs-Review** [comment]
Submitted by garjen55 on behalf of MOC Labs Review

**Comment by unaihuete93** [comment]
@MOC-Labs-Review Next time at least provide description and follow guidelines as mentioned in the PR template if possible [Collaboration Guide](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#pull-requests) thanks!

File: Instructions/Labs/AZ400_M05_Configuring_Agent_Pools_and_Understanding_Pipeline_Styles.md [hunk 1]
```diff
@@ -255,12 +256,12 @@ In this task, you will configure the LOD VM as an Azure DevOps self-hosting agen
...
 1.  On the **Configuring Agent Pools and Understanding Pipeline Styles** pane, in the vertical navigational pane on the left side, in the **Pipelines** section, click **Pipelines**. 
 1.  On the **Recent** tab of the **Pipelines** pane, select **PartsUnlimited** and, on the **PartsUnlimited** pane, select **Edit**.
-1.  On the **PartsUnlimited** edit pane, in the existing YAML-based pipeline, replace line **7** `vmImage: vs2017-win2016` designating the target agent pool the following content, designating the newly created self-hosted agent pool:
+1. On the **PartsUnlimited** edit pane, in the existing YAML-based pipeline, replace line **7** `vmImage: vs2017-win2016` with the following content, designating the newly created self-hosted agent pool:
 
     ```yaml
     name: az400m05l05a-pool
     demands:
-    - agent.name -equals az400m05-vm0
+    - agent.name -equals az400m05-vm0:
```
Comment (unaihuete93) [inline Instructions/Labs/AZ400_M05_Configuring_Agent_Pools_and_Understanding_Pipeline_Styles.md:264]:
```suggestion
    - agent.name -equals az400m05-vm0
```
not correct, it works without ":"

**Review by unaihuete93 — CHANGES_REQUESTED** [review]

**Comment by unaihuete93** [comment]
closing until feedback is received

**Pull request closed without merging**